### DISCUSSION TITLE
How to scrape JSON embeded in script tag at HTML?

### DISCUSSION THREAD

**Discussion by hkcomori** [FreshRSS/FreshRSS#6406] [Q&A]
I want to get the new posts from the JSON embedded in the following site.
Is that possible with FreshRSS?

I tried scraping with JSON dotpath but failed.

url: https://comic-walker.com/detail/KC_003631_S?episodeType=latest
id: `__NEXT_DATA__`

```html
<!-- some html -->
<script id="__NEXT_DATA__" type="application/json">
    {
        "props": {
            "pageProps": {
                "workId": "018d6a65-6bb2-79b0-8a26-d248741e2c22",
                "workCode": "KC_003631_S",
                "episodeType": "latest",
                "dehydratedState": {
                    "mutations": [
                    ],
                    "queries": [
...
</script>
<!-- some html -->
```

**Comment by Frenzie** [comment]
I don't think that's currently possible given that you'd first have to retrieve the JSON from that element.

**Reply by Alkarex** [reply]
Indeed, this is not something natively supported by FreshRSS at the moment, but options include making your own [FreshRSS extension](https://github.com/FreshRSS/Extensions/), or [RSS-Bridge](https://github.com/RSS-Bridge/rss-bridge) bridge.

**Reply by hkcomori** [reply]
Thank you. I still haven't achieved my goal, but I understand the approach.
It's necessary to extract the JSON by preprocessing with an extension or other tool before subscribing with FreshRSS's JSON dotpath, right?
Unfortunetly, I couldn't process it as JSON in FreshRSS because RSS-Bridge escapes the extracted results.
I will try to find another way.
